include/crocoddyl/core/activations/quadratic-flat-exp.hpp

```diff
@@ -0,0 +1,156 @@
+///////////////////////////////////////////////////////////////////////////////
+// BSD 3-Clause License
+//
+// Copyright (C) 2020, LAAS-CNRS, University of Edinburgh
+// Copyright note valid unless otherwise stated in individual files.
+// All rights reserved.
+///////////////////////////////////////////////////////////////////////////////
+
+#ifndef CROCODDYL_CORE_ACTIVATIONS_QUADRATIC_FLAT_EXP_HPP_
+#define CROCODDYL_CORE_ACTIVATIONS_QUADRATIC_FLAT_EXP_HPP_
+
+#include "crocoddyl/core/activation-base.hpp"
```

Comment:
The order of the header is not good. This is the proper order
```cpp
#include <iostream>
#include <stdexcept>
#include "crocoddyl/core/fwd.hpp"
 #include "crocoddyl/core/activation-base.hpp" 
#include "crocoddyl/core/utils/exception.hpp"
```

It is important to include first the system headers, and also the `fwd`